python/heterocl/tvm/runtime.py

```diff
@@ -47,12 +43,13 @@ def tvm_callback_exec_evaluate(platform, mode, host_only):
             print("[{}] Simulation runtime {}".format(
                 time.strftime("%H:%M:%S", time.gmtime()), runtime))
 
-        elif "csyn" in mode:
+        elif "csyn" in mode or mode == "customized":
```

Comment:
What is "customized" mode? Does it mean using the custom tcl scripts?

Reply:
Yes. If a custom Tcl script is used, the original mode should be discarded.